src/app/components/TransactionOverview/TransactionOverview.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import type { Transaction } from '../TransactionForm/TransactionForm'
+
+export type InitialStateType = {
```

Comment:
The name is confusing to me. Wouldn't it be better to say "TransactionProps" because it's about exactly that?

Reply:
thats a good idea. thanks